build-system/tasks/karma.conf.js

```diff
@@ -64,21 +68,13 @@ module.exports = {
   // Details: https://support.saucelabs.com/hc/en-us/articles/115010079868
   hostname: 'localhost',
 
+  babelifyConfig: BABELIFY_CONFIG,
```

Comment:
What's this for?

Reply:
It's used here to [reuse](https://github.com/ampproject/amphtml/blob/a76a1dc8a23c62ba739aaedad114541c521ccd67/build-system/tasks/runtime-test/runtime-test-base.js#L239-L242) the existing config instead of re-creating it.